pkg/p2p/kadcast/utils.go

```diff
@@ -88,6 +88,15 @@ func verifyIDNonce(id [16]byte, nonce [4]byte) error {
 	return errors.New("Id and Nonce are not valid parameters") //TODO: Create error type.
 }
 
+// Returns the ID associated to the chunk sent.
+// The ID is the half of the result of the hash of the chunk.
+func computeChunkID(chunk []byte) [16]byte {
+	var halfLenID [16]byte
+	fullID := sha3.Sum256(chunk)
```

Comment:
May be a good idea to use `blake2b` here, as it's more efficient and just as secure as `sha3`. I imagine we will be computing chunk IDs quite often.

Reply:
It's a really good idea. I will open an issue for it and we can take care once the functional implementation is finished.